packages/astro/src/@types/astro.ts

```diff
@@ -1412,34 +1412,51 @@ export interface AstroUserConfig {
 			/**
 			 * @docs
 			 * @name experimental.i18n.fallback
-			 * @type {Record<string, string[]>}
-			 * @version 3.*.*
+			 * @type {Record<string, string>}
+			 * @version 3.5.0
 			 * @description
 			 *
-			 * The fallback system of the locales. By default, the fallback system affect the **content only**, and it doesn't
-			 * do any redirects.
+			 * The content fallback strategy when navigating to pages in non-default languages that do not exist. (e.g. a translated page has not been created).
+			 *
+			 * This a plain object, where a key is a locale that hasn't traslated pages, and the corresponding value is the locale where you want to redirect the missing page to.
```

Comment:
```suggestion
			 * The content fallback strategy when navigating to pages in non-default languages that do not exist (e.g. a translated page has not been created).
			 *
			 * Use this object to declare a fallback `locale` for each language you support individually. If no fallback is specified, then all languages listed in `locales` will redirect the page to your `defaultLocale`.
```
This looks great! 

I made up that sentence at the end, because I think it helps to say what happens if you don't configure anything. 

I think a common thing to want to do here will be to make the one default fallback, and not have to list every language out individually. So, if the above is true, that's also very useful for doing this! If the above is NOT true, then I think we need to tell them how to set one default language for all other languages, as I expect this to be a common use case.

Reply:
The reason why I haven't put any default fallback is because:
- we do want to avoid redirects as much as possible **as default behaviour**
- it's possible that users don't require this feature (localised market, legal documents, etc.), so it would get more complicated to opt out of the default behaviour 

About this suggestion:

> If no fallback is specified, then all languages listed in `locales` will redirect the page to your `defaultLocale`.

As for now, we won't redirect to the `defaultLocale`, which means that if the user doesn't set a fallback for a locale, the missing pages will return 404.

We can explore a redirect behaviour later on

Reply:
Updated the comment (and the other one) with acceptable placeholder language that I'll check on! This is good enough to merge!